--- libconfig-1.3.2/Makefile.am.orig	2009-02-18 20:46:32.000000000 +0100
+++ libconfig-1.3.2/Makefile.am	2009-06-21 14:53:49.360487468 +0200
@@ -31,10 +31,10 @@
 libconfig___la_LDFLAGS = $(VERINFO) -no-undefined
 
 libsrc = libconfig.c scanner.l grammar.y private.h wincompat.h
-libinc = libconfig.h
+#libinc = libconfig.h
 
 libsrc_cpp =  $(libsrc) libconfigcpp.c++
-libinc_cpp =  $(libinc) libconfig.h++
+libinc_cpp =  libconfig.h libconfig.h++
 
 BUILT_SOURCES = scanner.c scanner.h grammar.c grammar.h
 
